dtsi.py

```diff
@@ -55,6 +101,7 @@ def generate_panel_dtsi(p: Panel, options: Options) -> None:
 	data-lanes = <{' '.join(map(str, p.lane_map.phys2log[:p.lanes]))}>;
 	remote-endpoint = <&panel_in>;
 }};
+
```

Comment:
similar here, avoid adding a trailing new line for something you generate conditionally later.